api_files/dependencies.json

```diff
@@ -120,6 +204,17 @@
                 ]
             }
         ],
+        "scenes[].action_mapping[].parameters.character_id": [
```

Comment:
Sorry-- this was unclear.  When I said, "if both character_id and a recipient parameter are provided, Warn that the recipient parameter is ignored", you mentally grouped "character_id" and "recipient".  But `character_id` is a property of the `action_mapping` and recipient is a `parameter`.

Anyway-- easy fix:  change to `"scenes[].action_mapping[].character_id"`

Reply:
Oh, thanks!